src/components/grid_overlay/grid_overlay.xml

```diff
@@ -10,6 +10,7 @@
       t-on-contextmenu.stop.prevent="onContextMenu">
       <FiguresContainer onFigureDeleted="props.onFigureDeleted"/>
       <DataValidationOverlay/>
+      <FilterIconsOverlay onMouseDown.bind="this.onMouseDown"/>
```

Comment:
```suggestion
      <FilterIconsOverlay onMouseDown.bind="(ev) => this.onMouseDown(ev, {closePopover: false})"/>
```
pass this version to the child component, it does not need to know about that specific option if it calls it with the same payload all the time